browser/resources/settings/brave_appearance_page/sidebar.html

```diff
@@ -1,21 +1,48 @@
 <style include="settings-shared iron-flex">
-  .settings-box.bottom-border {
-    border-top: none;
+  .border {
+    border-top: var(--cr-separator-line);
     border-bottom: var(--cr-separator-line);
   }
 </style>
-<div class="settings-box bottom-border">
-  <div class="flex" id="labelWrapper" aria-hidden="true">
-    <div class="label">$i18n{appearanceSettingsShowOptionTitle}</div>
-    <div class="secondary label" id="sub-label">
-      <span id="sub-label-text" aria-hidden="true">
-        [[sidebarShowEnabledLabel_]]
-      </span>
+
+<div class="cr-row">$i18n{sideBar}</div>
+<div class="list-frame">
+  <div class="cr-row list-item">
+    <div class="flex" id="labelWrapper" aria-hidden="true">
```

Comment:
just noticed we have `aria-hidden="true"` here and in some other places in this file - do you know why? Seems like we'd want it to be shown for screen readers.

Reply:
I think it's c&p error from my previous PR. Deleted.